config/owasp/suppressions.xml

```diff
@@ -60,4 +60,11 @@
     <gav regex="true">^io\.netty:netty-tcnative-boringssl-static:.*$</gav>
     <cpe>cpe:/a:netty_project:netty</cpe>
   </suppress>
+  <suppress>
+    <notes><![CDATA[file name: fasterxml:jackson-databind:2.9.9.1.jar -not applicable as
```

Comment:
This is not a fix, this is a workaround. Please update the jackson databind version in build.gradle

Reply:
yup :)